src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}
+      data-id={todo.id}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
Rewrite like this after destruction
```suggestion
      {user && <UserInfo user={user} />}
```